src/app/zap-templates/templates/app/ids/Attributes.zapt

```diff
@@ -1,51 +1,7 @@
 {{> header}}
-
 #pragma once
 
-#include <app/util/basic-types.h>
-
-namespace chip {
-namespace app {
-namespace Clusters {
-
-namespace Globals {
-namespace Attributes {
-
-{{#zcl_attributes_server}}
-{{#unless clusterRef}}
-namespace {{asUpperCamelCase label}} {
-static constexpr AttributeId Id = {{asMEI manufacturerCode code}};
-} // namespace {{asUpperCamelCase label}}
-
-{{/unless}}
-{{/zcl_attributes_server}}
-} // namespace Attributes
-} // namespace Globals
-
-
 {{#zcl_clusters}}
-{{#zcl_attributes_server}}
-{{#first}}
-namespace {{asUpperCamelCase parent.label}} {
-namespace Attributes {
-
-{{/first}}
-namespace {{asUpperCamelCase label}} {
-{{#if clusterRef}}
-static constexpr AttributeId Id = {{asMEI manufacturerCode code}};
-{{else}}
-static constexpr AttributeId Id = Globals::Attributes::{{asUpperCamelCase label}}::Id;
-{{/if}}
-} // namespace {{asUpperCamelCase label}}
-
-{{#last}}
-} // namespace Attributes
-} // namespace {{asUpperCamelCase parent.label}}
-
-{{/last}}
-{{/zcl_attributes_server}}
+#include <clusters/{{asUpperCamelCase label}}/AttributeIds.h>
```

Comment:
As long as we are naming new stuff, can we name the directories `{{asUpperCamelCase label preserveAcronyms=true}}` to better match the spec?  Yes, I know that won't match the namespace naming...

That said, I wonder whether we can create namespace aliases for these things in C++ too, to slowly start working toward the better names being usable and the incorrectly-cased ones being there just for compat?

Not sure this is worth blocking on, but renaming all these things later will be a bit of a pain is all.

Reply:
I would lean towards as-is to be able to have some consistency: both namespace and what can get code generation since space splitting is lost in https://github.com/project-chip/connectedhomeip/blob/master/src/app/zap-templates/templates/app/MatterIDL_Server.zapt#L21

Reply:
At a first glance it seems to me like darwin preserves acronyms and we carried over some IDL bits, from the look of it for bitmap/enum/event names ... this inconsistency is a pain :(

Reply:
Proposing any renames to be a followup: I cannot easily change the path and if we gather more mis-matches (cluster paths are already different, which is a pain) we get too much:
  - one name for `src/app/clusters/<name>`
  - another name for headers in `clusters/<name>/*.h` and class names
  - another name for namespaces (with aliases but still...) 
  
Sounds painful. We need some thought into this and how much we can preserve API compatibility (although we said we may break it once at 1.6 for the purpose of long term maintainability).